packages/skill-lesson/trpc/routers/pricing.ts

```diff
@@ -132,9 +130,11 @@ export const pricing = router({
 
       const token = await getToken({req: ctx.req})
 
+      const verifiedUserId = token?.sub
```

Comment:
I think it is generally going to be safer (and more accurate) to grab the `userId` off the authed JWT within the server-side code rather than relying on the client to pass in the relevant `userId`.